smart-contracts/test/UnlockDiscountToken/mintingTokens.js

```diff
@@ -13,7 +13,7 @@ let lock
 
 const estimateGas = 252166
 
-contract('UnlockDiscountToken (mainnet) / mintingTokens', (accounts) => {
```

Comment:
Please keep the `mainnet` indication because we do not have the same mechanism on side chains :)